pkgs/applications/networking/misc/zammad/default.nix

```diff
@@ -83,7 +82,7 @@ let
       };
       mini_racer = attrs: {
         buildFlags = [
-          "--with-v8-dir=\"${v8}\""
+          "--with-v8-dir=\"${nodejs.libv8}\""
```

Comment:
Likely breaking for cross but also, this is probably not cross-compiled typically.